common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-XmsPageableForGetListCalls_2023-09-21-23-34.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft.azure/openapi-validator-rulesets",
+      "comment": "Added XmsPageableForGetListCalls (RPC-Get-V1-13) rule to ensure that all GET LIST APIs contain the ‘x-ms-pageable’ property",
```

Comment:
>LIST [](http://example.com/codeflow?start=95&length=4)

can be lower case?

Reply:
No, cause the rule focus only on LIST calls, so mentioning in Caps makes more sense.